lapack/gonum/dgesc2.go

```diff
@@ -0,0 +1,57 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gonum
+
+import (
+	"math"
+
+	"gonum.org/v1/gonum/blas/blas64"
+)
+
+// Dgesc2 solves a system of linear equations
+//   A * X = scale* RHS
```

Comment:
Nit: clean up the equation, `//  A * X = scale * RHS` (two spaces between comment marks and text and single spaces around the operators).